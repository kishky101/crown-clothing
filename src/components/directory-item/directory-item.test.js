import renderer from 'react-test-renderer';
import DirectoryItem from './directory-item.component';
import { BrowserRouter as Router } from 'react-router-dom';

test('creating directory-item snapshot', () => {
    const directoryitem = renderer.create(
        <Router>
            <DirectoryItem category={{
                "id": 2,
                "title": "jackets",
                "imageUrl": "https://i.ibb.co/px2tCc3/jackets.png",
                route: 'shop/jackets'
            }} />
        </Router>
    ).toJSON;
    expect(directoryitem).toMatchSnapshot();
})