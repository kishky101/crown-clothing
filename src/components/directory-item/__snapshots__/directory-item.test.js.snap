// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating directory-item snapshot 1`] = `[Function]`;
